### ISSUE TITLE
CreatedAt is overrided by gorm even if explicitly set by caller.

### ISSUE THREAD

**Issue by anasanzari** [jinzhu/gorm#1801]
I'm trying to do this.
```
model.CreatedAt = created_at
db.Create(&model)

```
But the field is overrided with time.Now().

Is it possible to avoid this behavior?

**Comment by dima-kov** [comment]
Still actual

**Comment by iahmedov** [comment]
This creates really bad cases:
```
model.CreatedAt = now
model.DeletedAt = &now
```

but in database `DeletedAt` gets value which is earlier in time comparing to `CreatedAt`

**Comment by iahmedov** [comment]
For anyone who is encountered this problem, this is solved in newer versions, see 1.9.1

@jinzhu Close this issue please
